repository/src/main/java/org/apache/atlas/repository/store/graph/v2/preprocessor/PreProcessorUtils.java

```diff
@@ -198,4 +215,194 @@ public static void verifyDuplicateAssetByName(String typeName, String assetName,
             throw new AtlasBaseException(AtlasErrorCode.BAD_REQUEST, errorMessage);
         }
     }
+
+    public static void isValidLexoRank(String entityType, String input, String glossaryQualifiedName, String parentQualifiedName, EntityDiscoveryService discovery) throws AtlasBaseException {
+        // TODO : To remove this after migration is successful on all tenants and custom-sort is successfully GA
+        Pattern regex = Pattern.compile(LEXORANK_VALID_PATTERN);
```

Comment:
You are talking about Pattern regex right?
I am assuming its Pattern, so will be making that static

Reply:
I mean 
private static Pattern regex = Pattern.compile(LEXORANK_VALID_PATTERN); at class level, so that compile will be called only once & not for search request